include/pi-peps/linalg/arpack-rcdn.h

```diff
@@ -62,16 +63,20 @@ struct ARDNS {
 
   ARDNS(T& mmvp) : mvp(mmvp) {}
 
-  void real_nonsymm_runner(BLASINT const N,
+  void real_nonsymm(BLASINT const N,
                            BLASINT const nev,
                            BLASINT const max_ncv,
                            double const tol,
-                           BLASINT const maxIter) {
+                           BLASINT const maxIter,
+                           std::vector< std::complex<double> > & ev,
+                           std::vector<double> & V,
+                           bool rvec = false,
+                           bool dbg = false) {
     BLASINT ncv = std::max(max_ncv, 2 * nev + 1);
     BLASINT ldv = N;
     BLASINT lworkl = 3 * (ncv * ncv) + 6 * ncv;
 
-    std::cout << "N " << N << "\n"
+    if (dbg) std::cout << "N " << N << "\n"
```

Comment:
what is this `dbg`? what about putting a guard like 
```c++
#ifndef NDEBUG
std::cout <<..
#endif
```
such that when the library is in debug mode, it prints, and when compiled in release not? If you prefer, we might use another name, which can be set at build time, e.g. VERBOSE or set a verbosity level